icu4c/source/test/cintltst/cloctst.c

```diff
@@ -6750,67 +6750,79 @@ static void TestUsingDefaultWarning() {
     char cbuff[256];
     UErrorCode status = U_ZERO_ERROR;
     const char* language = "jJj";
+    const UChar* languageUChar = CharsToUChars("jjj");
```

Comment:
It seems like we'd want to remove the `const` here, as the CI pipelines fail due to discarding qualifiers.

Reply:
Oh, good point.
Hmm, I think I might actually change this to use `u_UCharsToChars` instead.